src/modules/routes/routes.js

```diff
@@ -0,0 +1,19 @@
+const Router = require('express');
```

Comment:
src/modules/routes/routes.js
duplication title
And so it is clear that the routes are in the routes folder.
Name them for example "task"